assembl/static2/js/app/components/administration/discussion/phaseForm.jsx

```diff
@@ -0,0 +1,147 @@
+// @flow
+import React from 'react';
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+import { connect } from 'react-redux';
+import { Translate, I18n } from 'react-redux-i18n';
+import { SplitButton, MenuItem } from 'react-bootstrap';
+import { Link } from 'react-router';
+import { type moment } from 'moment';
+import { modulesTranslationKeys } from '../../../constants';
+import { displayAlert } from '../../../utils/utilityManager';
+
+import { updatePhaseIdentifier, updateStartDate, updateEndDate } from '../../../actions/adminActions/timeline';
+
+type PhaseFormProps = {
+  phaseId: string,
+  phaseNumber: number,
+  identifier: string,
+  start: moment,
+  end: moment,
+  handleStartDateChange: Function,
+  handleEndDateChange: Function,
+  handleIdentifierChange: Function,
+  locale: string
+};
+
+export const DumbPhaseForm = ({
+  phaseId,
+  phaseNumber,
+  handleIdentifierChange,
+  handleStartDateChange,
+  handleEndDateChange,
+  identifier,
+  start,
+  end,
+  locale
+}: PhaseFormProps) => {
+  const onStartDateChange = (newStartDate) => {
+    if (newStartDate.isAfter(end)) {
+      displayAlert('danger', I18n.t('administration.timelineAdmin.startIsAfterEnd'));
+    } else {
+      handleStartDateChange(newStartDate);
+    }
+  };
+  const onEndDateChange = (newEndDate) => {
+    if (!newEndDate.isAfter(start)) {
+      displayAlert('danger', I18n.t('administration.timelineAdmin.endIsBeforeStart'));
+    } else {
+      handleEndDateChange(newEndDate);
+    }
+  };
+
+  const startDatePickerPlaceholder = I18n.t('administration.timelineAdmin.selectStart', { count: phaseNumber });
+  const endDatePickerPlaceholder = I18n.t('administration.timelineAdmin.selectEnd', { count: phaseNumber });
+
+  const splitButtonTitle = identifier ?
+    I18n.t(`administration.modules.${identifier}`) :
+    I18n.t('administration.timelineAdmin.singleModule');
+
+  return (
+    <div className="phase-form">
+      <div className="date-picker-field">
```

Comment:
do we want to create a `DatePickerField` component for this whole div ? (with a storybook for it of course)